proposal.html

```diff
@@ -25,8 +25,8 @@ <h1>Array.prototype.flatMap ( _mapperFunction_ [ , _thisArg_ ] )</h1>
   </emu-alg>
 </emu-clause>
 <emu-clause id="sec-Array.prototype.flatten">
-  <h1>Array.prototype.flatten( [ _depth_ ] )</h1>
-  <p>When the `flatten` method is called with zero or one arguments, the following steps are taken:</p>
+  <h1>Array.prototype.flatten</h1>
+  <p>`Array.prototype.flatten` is an accessor property. Its get accessor function returns a function named `"flatten"` with length *0* which, when called with parameter _depth_, performs the following steps:</p>
```

Comment:
I don't think this is the right way to specify this. For example, it leaves ambiguous whether the function so returned has the same identity each time it's retrieved, or across realms.

Instead I think you want to define a new [intrinsic object](https://tc39.github.io/ecma262/#table-7) for the function itself, say `%ArrayProto_flatten_value%`, and have the behavior of the accessor be `1. Return %ArrayProto_flatten_value%` or whatever.

Reply:
Yeah, you're probably right. I was trying to avoid having to do exactly this.